hbase-store/src/main/resources/META-INF/services/geotrellis.spark.io.AttributeStoreProvider

```diff
@@ -0,0 +1 @@
+geotrellis.spark.io.hbase.HBaseLayerProvider
```

Comment:
These file names need to be changed to the file path `geotrellis.layers.AttributeStoreProvider`.

Also, the name of the HBase provider would be: `geotrellis.store.hbase.HBaseCollectionLayerProvider`

Reply:
This pertains to all of the other service files as well.